src/components/parameter-tab.tsx

```diff
@@ -0,0 +1,443 @@
+                aria-label="parameters"
+                orientation="vertical"
+                sx={styles.listDisplay}
+            >
+                <Box sx={styles.listTitleDisplay}>
+                    <FormattedMessage id="parameters" />
+                </Box>
+                <Tab
+                    sx={styles.listItemDisplay}
+                    disabled={!props.studyId}
+                    label={<FormattedMessage id="LoadFlow" />}
+                    value={TAB_VALUES.lfParamsTabValue}
+                />
+                <Tab
+                    sx={styles.listItemDisplay}
+                    disabled={
+                        !(
+                            getStudyUrlWithNodeUuid ||
+                            securityAnalysisAvailability ===
+                                OptionalServicesStatus.Up
+                        )
+                    }
+                    label={<FormattedMessage id="SecurityAnalysis" />}
+                    value={TAB_VALUES.securityAnalysisParamsTabValue}
+                />
+                <Tab
+                    sx={styles.listItemDisplay}
+                    disabled={
+                        !(
+                            getStudyUrlWithNodeUuid ||
+                            sensitivityAnalysisAvailability ===
+                                OptionalServicesStatus.Up
+                        )
+                    }
+                    label={<FormattedMessage id="SensitivityAnalysis" />}
+                    value={TAB_VALUES.sensitivityAnalysisParamsTabValue}
+                />
+                <Tab
+                    sx={styles.listItemDisplay}
+                    disabled={
+                        !(
+                            getStudyUrlWithNodeUuid ||
+                            shortCircuitAvailability ===
+                                OptionalServicesStatus.Up
+                        )
+                    }
+                    label={<FormattedMessage id="ShortCircuit" />}
+                    value={TAB_VALUES.shortCircuitParamsTabValue}
+                />
+                <Tab
+                    sx={styles.listItemDisplay}
+                    disabled={
+                        !(
+                            getStudyUrlWithNodeUuid ||
+                            dynamicSimulationAvailability ===
+                                OptionalServicesStatus.Up
+                        )
+                    }
+                    label={<FormattedMessage id="DynamicSimulation" />}
+                    value={TAB_VALUES.dynamicSimulationParamsTabValue}
+                />
+                <Tab
+                    sx={styles.listItemDisplay}
+                    disabled={
+                        !(
+                            getStudyUrlWithNodeUuid ||
+                            voltageInitAvailability ===
+                                OptionalServicesStatus.Up
+                        )
+                    }
```

Comment:
if I understand correctly from [#be0896f](//github.com/gridsuite/gridstudy-app/pull/1768/commits/be0896f59327c5b3c1c9db02f393566227f27371), it would be:
```suggestion
                    disabled={
                        !props.studyId ||
                        voltageInitAvailability !==
                            OptionalServicesStatus.Up
                    }
```